beta/src/components/MDX/Sandpack/index.tsx

```diff
@@ -46,20 +46,20 @@ const SandpackGlimmer = ({code}: {code: string}) => (
 );
 
 export default memo(function SandpackWrapper(props: any): any {
-  const codeSnippet = createFileMap(Children.toArray(props.children));
+  const codeSnippets = Children.toArray(props.children) as React.ReactElement[];
```

Comment:
Using same pattern as `beta/src/components/MDX/Sandpack/SandpackRoot.tsx`. Should we pass `files` as a prop instead of recomputing in `beta/src/components/MDX/Sandpack/SandpackRoot.tsx`?